### PR TITLE
Send rent expiry email

### PR DESCRIPTION
**Pull request by aleochoam**
### Summary
- Added the gem sidekiq-scheduler
- Created the job for seeking the expiring rents and mail each one
- Settting the current time zone (Buenos Aires)
- Added the I18n translations and the layouts for the email
- Cron the job to be executed at 9:00 everyday

### CODE REVIEW

File: app/mailers/rent_mailer.rb [hunk 1]
```diff
@@ -5,6 +5,15 @@ def rent_confirmation
     @user = @rent.user
     @book = @rent.book
 
-    mail(to: @user.email, subject: 'Rent Confirmation Email')
+    mail(to: @user.email, subject: t('.title', title: @book.title))
+  end
+
+  def rent_expiry
+    rent_id = params[:rent_id]
```
Comment (Gmarn) [inline app/mailers/rent_mailer.rb]: This line is not very useful.

File: app/mailers/rent_mailer.rb [hunk 2]
```diff
@@ -5,6 +5,15 @@ def rent_confirmation
...
+    @rent = Rent.find(rent_id)
```
Comment (Gmarn) [inline app/mailers/rent_mailer.rb]: You could put this in a method with memorization
```ruby
def rent
  @rent ||= Rent.find(params[:rent_id]
end
```
And call it in this mailer method and the one above.

File: config/application.rb [hunk 3]
```diff
@@ -19,5 +19,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     I18n.available_locales = [:en, :es]
+
+    config.time_zone = 'Buenos Aires'
```
Comment (Gmarn) [inline config/application.rb]: Better to not change this for all you app, but only on your cron.

File: config/sidekiq.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+:schedule:
+  rent_validation:
+    cron: '0 9 * * *'
```
Comment (Gmarn) [inline config/sidekiq.yml]: You can define it directly here, as ths example:
`cron: '0 0 * * * America/Santiago'`

Or [Here](https://stackoverflow.com/questions/13289751/cron-job-in-a-different-timezone) there is some more help.

File: app/views/rent_mailer/rent_expiry.html.slim [hunk 5]
```diff
@@ -0,0 +1,8 @@
+h1
+  = t('.title' , title: @book.title)
```
Comment (Gmarn) [inline app/views/rent_mailer/rent_expiry.html.slim:2]: It's strange, that you repeat the `title` here...

File: app/jobs/rent_validation_job.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class RentValidationJob < ApplicationJob
```
Comment (Gmarn) [inline app/jobs/rent_validation_job.rb]: We prefer use `/app/workers/` folder at wolox. Also you can delete `ApplicationJob`, no usefull here

Comment (Gmarn) [reply app/jobs/rent_validation_job.rb]: And we name workers as: `RentValidationWorker`

**Review by Gmarn — CHANGES_REQUESTED** [review]

File: app/views/rent_mailer/rent_expiry.html.slim (hunk 5 above)
Comment (aleochoam) [reply app/views/rent_mailer/rent_expiry.html.slim:2]: What do you mean by 'repeat'?

Comment (Gmarn) [reply app/views/rent_mailer/rent_expiry.html.slim:2]: You ever put this on the subject of the mail, so it seems strange to me that you repeat it here.

**Review by Gmarn — APPROVED** [review]

**Pull request merged**